### ISSUE TITLE
Exporting 2015 data

### ISSUE THREAD

**Issue by xavriley** [atiro/PFIDB#2]
Hi @atiro and @jms301 - hope you're both well!

I had a quick look at this again today as it would be nice to update this ahead of the next AccHack in just over a week's time.

Using the projects.json I added in the last commit I tried importing that into my local pfi-explorer but I'm getting this:

```
$ mongoimport -h 127.0.0.1:27017 -d meteor -c projects ~/Projects/PFIDB/projects.json
2015-11-11T15:33:22.515+0000    connected to: 127.0.0.1:27017
2015-11-11T15:33:22.524+0000    Failed: error unmarshaling bytes on document #0: JSON decoder out of sync - data changing underfoot?
2015-11-11T15:33:22.524+0000    imported 0 documents
```

Looking at my data it seems I just exported an array of arrays which probably isn't the right structure for the import.

TLDR: Do either of you know how to produce the right format for `projects_load.json`  e.g. https://github.com/jms301/pfi-explorer/blob/master/.data/projects_load.json

Is there a step that I'm missing?

**Comment by jms301** [comment]
Hey Xav,

It's all rather fuzzy at this point but I'm pretty sure I used jq to modify
the files into the right shape.

I think you want a file with one object per line, the abandoned PFI-E v2
had scripts to do this see:
https://github.com/jms301/audit-thingie/tree/master/.data
The contracts.json file is similar to the projects.json in PFI-E v1 but it
had a lot of the arrays normalized out, perhaps the scripts used to
generate that will be helpful.

James
